packages/x-date-pickers/src/locales/deDE.ts

```diff
@@ -23,18 +33,35 @@ const deDEPickers: Partial<PickersLocaleText<any>> = {
   todayButtonLabel: 'Heute',
 
   // Clock labels
-  // clockLabelText: (view, time, adapter) => `Select ${view}. ${time === null ? 'No time selected' : `Selected time is ${adapter.format(time, 'fullTime')}`}`,
-  // hoursClockNumberText: hours => `${hours} hours`,
-  // minutesClockNumberText: minutes => `${minutes} minutes`,
-  // secondsClockNumberText: seconds => `${seconds} seconds`,
+  clockLabelText: (view, time, adapter) =>
+    `${views[view] ?? view} auswählen. ${
+      time === null
+        ? 'Keine Uhrzeit ausgewählt'
+        : `Ausgewählte Uhrzeit ist ${adapter.format(time, 'fullTime')}`
```

Comment:
To keep consistency? (with the other suggestions)
```suggestion
        : `Gewählte Uhrzeit ist ${adapter.format(time, 'fullTime')}`
```

Reply:
See latest commit.